jax/_src/numpy/lax_numpy.py

```diff
@@ -397,23 +397,132 @@ def _conv(x: Array, y: Array, mode: str, op: str, precision: PrecisionLike,
   return result[0, 0, out_order]
 
 
-@util.implements(np.convolve, lax_description=_PRECISION_DOC,
-                 extra_params=_CONV_PREFERRED_ELEMENT_TYPE_DESCRIPTION)
 @partial(jit, static_argnames=('mode', 'precision', 'preferred_element_type'))
 def convolve(a: ArrayLike, v: ArrayLike, mode: str = 'full', *,
              precision: PrecisionLike = None,
              preferred_element_type: DTypeLike | None = None) -> Array:
+  """Convolution of two one dimensional arrays.
+
+  JAX implementation of :func:`numpy.convolve`.
```

Comment:
I'm putting myself in the user's shoes, and I think I'd still be confused about what this does (particularly in contrast to `jnp.correlate`).

Maybe we should add a mathematical definition, something like
```
$c_k = \sum_j a_{k - j} v_j$
```
which renders as

$$
c_k = \sum_j a_{k - j} v_j
$$

I think this is the correct expression for the full 1D convolution.

The expression for correlation is similar, I think, except it's `k + j` and `v` is conjugated.

That would make it more clear at a glance what the relationship is between the two functions.

What do you think?

Reply:
Yes, providing mathematical definition would be great in user point of view. Included the definition. Thanks @jakevdp.

Reply:
Also included example for complex arrays.

Reply:
I have a doubt on the type of inputs. Does they need to be `Array` or `ArrayLike`?

Reply:
Inputs should be `ArrayLike`, which does not include lists.

Reply:
Understood. Thanks for clarifying.